src/web/package.json

```diff
@@ -41,7 +43,8 @@
     "@types/node": "16.11.26",
     "@types/react": "17.0.39",
     "babel-loader": "8.2.3",
+    "eslint": "7.32.0",
     "terser-webpack-plugin": "5.3.1",
-    "typescript": "4.5.5"
+    "typescript": "4.4.4"
```

Comment:
Why are we downgrading the typescript version?

Reply:
I was hitting an error with TypeScript and it wanted TypeScript >=3.5 && <=4.5. I tried reverting this change and now it's happy with TypeScript 4.5.3

Reply:
Nevermind, it's still there @_@ 
```
> @senecacdot/satellite@1.26.0 lint C:\Users\lecin\Documents\repo\telescope\src\satellite
> pnpm eslint


> @senecacdot/satellite@1.26.0 eslint C:\Users\lecin\Documents\repo\telescope\src\satellite

=============

WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.

You may find that it works just fine, or you may not.

SUPPORTED TYPESCRIPT VERSIONS: >=3.3.1 <4.5.0

YOUR TYPESCRIPT VERSION: 4.5.5

Please only submit bug reports when using the officially supported version. 
```